pkg/ultradns/ultradns.go

```diff
@@ -66,8 +67,8 @@ func NewAPIConnection(options *APIOptions) *APIConnection {
 // * encountering an error authorizing
 // * Failing to connect to the API server
 // * When getting an HTTP status code of >= 400
-func (apiConn *APIConnection) Get(url string) (*http.Response, error) {
-	if err := apiConn.Authorization.Authorize(apiConn.Client); err != nil {
+func (apiConn *APIConnection) Get(url string) (resp *http.Response, err error) {
```

Comment:
TIL - You can initialize the return variables in the return types here.  Very cool!